Guides/Development.md

```diff
@@ -116,7 +120,6 @@ If you have a setup for developing the driver in an editor other than the ones l
 ## Workflow
 1. Create a feature branch, named by the corresponding JIRA ticket if exists, along with a short descriptor of the work: for example, `SWIFT-30/writeconcern`.
 1. Do your work on the branch.
-1. If you add, remove, or rename any tests, make sure to update `LinuxMain.swift` accordingly. If you are on MacOS, you can do that by running `make linuxmain`.
 1. Ensure your code passes both the linter and the formatter.
 1. Make sure your code builds and passes all tests on:
     - [Travis](https://travis-ci.org/mongodb/mongo-swift-driver). Every time you push to GitHub or open a pull request, it will trigger a new build, which includes running the linter, formatter, and basic tests.
```

Comment:
can you remove this line please? we no longer test on travis, only evergreen

Reply:
what are people who dont work at mongodb supposed to do? can they run evergreen?

Reply:
if they make a PR, one of us is able to trigger testing it on evergreen. the PR status shows something like this
![Screen Shot 2022-01-06 at 5 55 39 PM](https://user-images.githubusercontent.com/5976377/148463979-02775ec0-afce-44f8-bdba-44e3311327ea.png)

and then if one of us clicks that link it takes you to a page to manually authorize the patch.

so you could say something like "ask a maintainer to trigger a test run on evergreen for you"